jwql/website/apps/jwql/views.py

```diff
@@ -342,7 +340,7 @@ def archive_thumbnails_ajax(request, inst, proposal, observation=None):
     inst = JWST_INSTRUMENT_NAMES_MIXEDCASE[inst.lower()]
 
     data = thumbnails_ajax(inst, proposal, obs_num=observation)
-
+    data['thumbnail_sort'] = request.session.get("image_sort", "Ascending")
```

Comment:
@BradleySappington is this an example of storing session data like we discussed in the past? Also, does this set the default for the thumbails to be in Ascending order?

Reply:
Yes and Yes